### PR TITLE
build: fetch img tool from upstreamartifacts storage

### PR DESCRIPTION
**Pull request by mboersma**
**Reason for Change**:
Moves the download source for the `img` utility to our preferred production blob storage.

This is the first baby step in an effort to move off of the `acs-mirror` blob storage (and replace manual steps with automated pipelines).

**Issue Fixed**:
Refs #2545

**Requirements**:

- [x] uses [conventional commit messages](https://www.conventionalcommits.org/)
  
- [ ] includes documentation
- [ ] adds unit tests
- [ ] tested upgrade from previous version

**Notes**:

### CODE REVIEW

File: parts/k8s/cloud-init/artifacts/cse_install.sh [hunk 1]
```diff
@@ -243,7 +243,7 @@ installContainerd() {
 
 installImg() {
     img_filepath=/usr/local/bin/img
-    retrycmd_get_executable 120 5 $img_filepath "https://acs-mirror.azureedge.net/img/img-linux-amd64-v0.5.6" ls || exit $ERR_IMG_DOWNLOAD_TIMEOUT
```
Comment (jchauncey) [inline parts/k8s/cloud-init/artifacts/cse_install.sh]: Does this have a CDN in front of it? I wonder if we should do that for the new url too...

Comment (mboersma) [reply parts/k8s/cloud-init/artifacts/cse_install.sh]: /hold

Yes acs-mirror.azureedge.net is a CDN setup, so we should probably do similarly for the new storage. No rush on this of course.

Comment (jchauncey) [reply parts/k8s/cloud-init/artifacts/cse_install.sh]: ok cdn created - 

```
╰ wget https://upstreamartifacts.azureedge.net/img/img-linux-amd64-v0.5.6               ado-admin/default ⎈     03:57:03  
--2020-01-16 15:57:49--  https://upstreamartifacts.azureedge.net/img/img-linux-amd64-v0.5.6
Resolving upstreamartifacts.azureedge.net (upstreamartifacts.azureedge.net)... 13.107.246.10, 2620:1ec:bdf::10
Connecting to upstreamartifacts.azureedge.net (upstreamartifacts.azureedge.net)|13.107.246.10|:443... connected.
HTTP request sent, awaiting response... 200 OK
Length: 25700016 (25M) [application/octet-stream]
Saving to: ‘img-linux-amd64-v0.5.6’

img-linux-amd64-v0.5.6           100%[=======================================================>]  24.51M  1.74MB/s    in 17s     

2020-01-16 15:58:08 (1.42 MB/s) - ‘img-linux-amd64-v0.5.6’ saved [25700016/25700016]
```

Comment (mboersma) [reply parts/k8s/cloud-init/artifacts/cse_install.sh]: Thanks @jchauncey! I updated to reference that CDN-fronted URL.

/hold cancel

Comment (mboersma) [reply parts/k8s/cloud-init/artifacts/cse_install.sh]: /hold cancel

**Comment by mboersma** [comment]
`Path does not exist: /home/vsts/work/1/s/_logs`: Azure Pipelines is misbehaving, but CI has already proved this single URL change works, so I'm merging it.

**Pull request merged**